docs/Performance-Testing.md

```diff
@@ -0,0 +1,65 @@
+The configuration described below is designed for a performance testing use case; however, it may be a generally useful starting point for other use cases.
+
+This setup establishes the following four services:
+- Fedora
+- PostgreSQL
+- Prometheus
+- Grafana
+
+In this setup Fedora uses PostgreSQL as its rebuildable backend database.
+Prometheus and Granfana are strictly used for collecting and displaying Fedora performance status, and can safely be removed if unwanted.
+
+After starting these services, Fedora will be available at: http://host:8080/fcrepo/rest
+- default username / password: fedoraAdmin / fedoraAdmin
+Grafana will be available at: http://host:3000
+- default username / password: admin / admin
+
+# System Setup
+
+1. [Install Docker](https://docs.docker.com/get-docker/)
+1. Create a Fedora home directory for all repository persistence. This configuration expects:
+   ```
+   sudo mkdir -p /opt/fcrepo-home
```

Comment:
I'm not sure if this applies to other operating systems, but on Mac OS I had to add /opt to the list of directories that can be mounted into Docker by opening the Docker application Preferences and navigating to Resources -> File Sharing. After adding /opt I hit Apply & Restart.